File: TransCoder_tokenized_test_set_functions/python/MINIMUM_PRODUCT_K_INTEGERS_ARRAY_POSITIVE_INTEGERS.py
def minProduct ( arr, n, k ) :
    heapq . heapify ( arr )
    count = 0
    ans = 1
    while ( arr ) and count < k :
        x = heapq . heappop ( arr )
        ans = ans * x
        count = count + 1
    return ans;

File: TransCoder_tokenized_test_set_functions/java/MINIMUM_PRODUCT_K_INTEGERS_ARRAY_POSITIVE_INTEGERS.java
public static int minProduct ( int [ ] arr, int n, int k ) {
  PriorityQueue < Integer > pq = new PriorityQueue < > ( );
  for ( int i = 0;
  i < n;
  i ++ ) pq . add ( arr [ i ] );
  int count = 0, ans = 1;
  while ( pq . isEmpty ( ) == false && count < k ) {
    ans = ans * pq . element ( );
    pq . remove ( );
    count ++;
  }
  return ans;
}